packages/compass-preferences-model/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import Preferences from './preferences';
+
+export default Preferences;
```

Comment:
Nice, you already did the TS conversion! :blue_heart:

Reply:
Yeah decided to do it sooner than later to catch potential issues since it touches so many parts of the app.